mod/05_standard/set/self/account_links.rb

```diff
@@ -5,8 +5,9 @@
     #ENGLISH
     links = []
     if Auth.signed_in?
-      links << link_to_page( Auth.current.name, nil, :id=>'my-card-link' )
+      links << build_link( Auth.current.cardname, nil, :id=>'my-card-link' )
```

Comment:
When we start requiring ruby 2+, we can address that with optional keyword arguments.  The nil is typically for the link text argument that is optional, but because the next positional argument is used it needs to be explicit.